doc/release/release_dev.rst

```diff
@@ -56,18 +56,38 @@ X contributors. Highlights include:
   handling multigraph weights, such as ``mean`` or ``median``. Finally, this
   also includes support for structured dtypes, which enables the creation of
   multi-attribute adjacency matrices and replaces the less generic
-  ``to_numpy_recarray``.
-
-**Maybes**
+  `~networkx.convert_matrix.to_numpy_recarray`.
```

Comment:
I generally reduce the amount of linking in the release notes, especially for deprecated functions, as this will cause a broken link in the future when the function is removed. It's not really important though!